Update Autograd Export Docs (#56594)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #59539 Add new_ones symbolic (#59255)
* #59538 Symbolic for ReLu6 (#58560)
* #59537 [ONNX] Sum empty tensor could not be exported to ONNX successfully. (#58141)
* #59536 [ONNX] Add support for torch.bernoulli() export (#57003)
* #59535 fix error message for type_as (#57948)
* **#59534 Update Autograd Export Docs (#56594)**

Update autograd export docs

Co-authored-by: neginraoof <neginmr@utexas.edu>

Differential Revision: [D29046606](https://our.internmc.facebook.com/intern/diff/D29046606)

neginraoof committed:
Merge (#56594)

[ghstack-poisoned]

BowenBao committed:
Update on "Update Autograd Export Docs (#56594)"


Update autograd export docs

Co-authored-by: neginraoof <neginmr@utexas.edu>



[ghstack-poisoned]

SplitInfinity commented:
@splitinfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29046606).

Commit:
Update on "Update Autograd Export Docs (#56594)"


Update autograd export docs

Co-authored-by: neginraoof <neginmr@utexas.edu>

Differential Revision: [D29046606](https://our.internmc.facebook.com/intern/diff/D29046606)

[ghstack-poisoned]
